tools/bin/ci_check_dependency.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import os
+import os.path
+
+CONNECTOR_FOLDER = "airbyte-integrations/connectors/"
+CONNECTOR_PATH = "./airbyte-integrations/connectors/"
+WHITE_LIST = ["/src/main/", "build.gradle"]
+
+
+def main():
+    git_diff_file_path = ' '.join(sys.argv[1:])
+
+    # Get changed connectors files
+    connectors_changed_files = get_connectors_changed_files(git_diff_file_path)
```

Comment:
If I am reading this correctly, this approach will miss changes in `airbyte-integrations/bases`, which are the most likely ones to affect many connectors

Reply:
Probably I miss understand the description of the task. But it's really nice catch I'll update it